rewrite-java/src/main/java/org/openrewrite/java/cleanup/RemoveInstanceOfPatternMatch.java

```diff
@@ -156,6 +144,7 @@ public J.If visitIf(J.If iff, ExecutionContext executionContext) {
                                     J.Block.createEmptyBlock().withStatements(
                                             Collections.singletonList(elsePart.getBody())))),
                             executionContext);
+                    elsePart = result.getElsePart();
```

Comment:
The statement above replaces the else part and since the `elsePart` variable is dereferenced on line 153 again, it needs to be updated here.